README.md

```diff
@@ -9,24 +11,60 @@ relation.
 
 ```javascript
 function mystery(n) {
-    if(n <= 1)
+    if(n <= 1)            O(1)
         return;
     else {
-        mystery(n / 3);
-        var count = 0;
-        mystery(n / 3);
-        for(var i = 0; i < n*n; i++) {
-            for(var j = 0; j < n; j++) {
-                for(var k = 0; k < n*n; k++) {
-                    count = count + 1;
+        mystery(n / 3);                           n/3
+        var count = 0;                            O(1)
+        mystery(n / 3);                           n/3
+        for(var i = 0; i < n*n; i++) {            n^2
+            for(var j = 0; j < n; j++) {          n
+                for(var k = 0; k < n*n; k++) {    n^2
+                    count = count + 1;            O(1)
                 }
             }
         }
-        mystery(n / 3);
+        mystery(n / 3);                           n/3
     }
 }
 ```
 
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+
+$$
+T(n) = 
+\begin{cases} 
+1 & \text{for } n \leq 1 \\ 
+3\left(T\left(\frac{n}{3}\right) + n^{5}\right) & \text{for } n > 1 
+\end{cases}
+$$
+
+$$ T(n) = 3\left(T\left(\frac{n}{3}\right) + n^{5}\right) $$
+
+$$ T(n) = 3\left(3\left(T\left(\frac{n}{9}\right) + n^{5}\right)\right) $$
+
+$$ T(n) = 9\left(T\left(\frac{n}{9}\right) + n^{5}\right) $$
```

Comment:
With every expansion, you're getting a new $+ n^5$ term -- you're missing that here.